applications/materialDataCreator/materialDataCreator.html

```diff
@@ -0,0 +1,573 @@
+                }
+                if (material.opacityMap === "USE") {
+                    materialData += '            "opacityMap": "' + material.albedoMap.escapeJSON() + '",' + "\n";
+                }  
+                if (material.emissiveMap !== "") {
+                    materialData += '            "emissiveMap": "' + material.emissiveMap.escapeJSON() + '",' + "\n";
+                }                 
+                if (material.scatteringMap !== "") {
+                    materialData += '            "scatteringMap": "' + material.scatteringMap.escapeJSON() + '",' + "\n";
+                }
+                if (material.occlusionMap !== "") {
+                    materialData += '            "occlusionMap": "' + material.occlusionMap.escapeJSON() + '",' + "\n";
+                }                
+                materialData += '            "model": "hifi_pbr"' + "\n";
+                materialData += '        }' + "\n";
+                materialData += '    ]' + "\n";
+                materialData += '}';
+                
+                document.materialEditor.output.value = materialData;
+                var message = {
+                    "chanel": chanel,
+                    "action": "updateMaterialData",
+                    "materialData": materialData,
+                    "materialRecord": material
+                };
+                
+                EventBridge.emitWebEvent(JSON.stringify(message));
+                
+            }
+
+            String.prototype.escapeJSON = function() {
+                var result = "";
+                for (var i = 0; i < this.length; i++) {
+                    var ch = this[i];
+                    switch (ch)
+                    {
+                        case "\\": ch = "\\\\"; break;
+                        //case "\'": ch = "\\'"; break;
+                        case "\"": ch = '\\"'; break;
+                        //case "\&": ch = "\\&"; break;
+                        case "\t": ch = "\\t"; break;
+                        case "\n": ch = "\\n"; break;
+                        case "\r": ch = "\\r"; break;
+                        case "\b": ch = "\\b"; break;
+                        case "\f": ch = "\\f"; break;
+                        case "\v": ch = "\\v"; break;
+                        default: break;
+                    }
+                    
+                    result += ch;
+                }
+                
+                return result;
+            };
+            
+            var chanel = "materialDataCreator.ak.vircadia";
+
+
+			function copyToClipboard(data) {
+				var $temp = $("<input>");
+				$("body").append($temp);
+				$temp.val(data).select();
+				document.execCommand("copy");
+				$temp.remove();
+			}
+            
+            function demo(shape) {
+                var message = {
+                    "chanel": chanel,

```

Comment:
Is this supposed to mean channel?
```suggestion
                    "channel": channel,
```